modules/rosa-sdpolicy-networking.adoc

```diff
@@ -0,0 +1,65 @@
+
+== Custom certificate authorities for builds
+{product-title} supports the use of custom certificate authorities to be trusted by builds when pulling images from an image registry.
+
+== Load Balancers
+{product-title} uses up to five different load balancers:
+
+- Internal master load balancer that is internal to the cluster and used to balance traffic for internal cluster communications.
+- External master load balancer that is used for accessing the OpenShift and Kubernetes APIs. This load balancer can be disabled in OCM. If this load balancer is disabled, Red Hat reconfigures the API DNS to point to the internal master load balancer.
+- External master load balancer for Red Hat that is reserved for cluster management by Red Hat. Access is strictly controlled, and communication is only possible from whitelisted bastion hosts.
+- Default external router/ingress load balancer that is the default application load balancer, denoted by apps in the URL. The default load balancer can be configured in OCM to be either publicly accessible over the Internet or only privately accessible over a pre-existing private connection. All application routes on the cluster are exposed on this default router load balancer, including cluster services such as the logging UI, metrics API, and registry.
+- Optional: secondary router/ingress load balancer that is a secondary application load balancer, denoted by apps2 in the URL. The secondary load balancer can be configured in OCM to be either publicly accessible over the Internet or only privately accessible over a pre-existing private connection. If a `Label match` is configured for this router load balancer, then only application routes matching this label are exposed on this router load balancer, otherwise all application routes are also exposed on this router load balancer.
+- Optional: Load balancers for Services enable non-HTTP/SNI traffic and non-standard ports for services.  These load balancers can be mapped to a service running on {product-title} to enable advanced ingress features, such as non-HTTP/SNI traffic or the use of non-standard ports. These can be purchased in groups of 4 for standard clusters or can be provisioned without charge in CCS clusters, however each AWS account has a quota which link:https://docs.aws.amazon.com/elasticloadbalancing/latest/classic/elb-limits.html[limits the number of Classic Load Balancers] that can be used within each cluster.
+
+== Cluster ingress
+Project administrators can add route annotations for many different purposes, including ingress control through IP whitelisting.
+
+Ingress policies can also be changed by using NetworkPolicy objects, which leverage the `ovs-networkpolicy` plugin. This allows for full control over the ingress network policy down to the pod level, including between pods on the same cluster and even in the same namespace.
```

Comment:
s/NetworkPolicy objects/`NetworkPolicy` objects
s/plugin/plug-in

Reply:
Done.